internal/cli/kraft/net/create/create.go

```diff
@@ -20,6 +20,13 @@ import (
 	"kraftkit.sh/machine/network"
 )
 
+const (
+	// defaultNetworkMaxSize is the maximum size of a network name on Linux.
+	// We subtract 2 to account for the network interface suffix.
+	// The value 16 is taken from the glibc headers.
+	defaultNetworkNameMaxSize = 15 - 2
```

Comment:
This is still a somewhat tight limit. We append at least 4 characters, possibly going into double digits or maybe more (`@if%d`). A safe limit could be obtained by looking at the mask size of the network and seeing the number of interfaces that we could have on that network, but this limit might be too loose for the average use case and prevent users from naming their network properly.

Maybe we could have this limit (maybe rather 15 - 4) and warn the users about not being able to fully use the address space, possibly erroring in a better way if we run out of names for interfaces?

Reply:
Ah sorry I thought it was just a number, then I'll decrease to accomodate for `@if%d%d`

Reply:
As for erroring in the code when out of bounds makes sense, though I can't fully point the place where this should happen. Maybe you can do this in a separate PR? 100 interfaces should be enough for now :D